modules/apps/headless/headless-discovery/headless-discovery-web/package.json

```diff
@@ -10,8 +10,14 @@
 		"@clayui/tabs": "3.1.1",
 		"frontend-js-react-web": "*",
 		"frontend-js-web": "*",
+		"graphiql": "0.14.2",
+		"graphql": "14.0.0",
 		"react": "16.12.0"
 	},
+	"devDependencies": {
+		"liferay-npm-bundler-loader-css-loader": "2.18.4",
+		"liferay-npm-bundler-loader-json-loader": "2.18.4"
```

Comment:
Let's follow up to remove these because [we don't want devDependencies to go in this way](https://github.com/liferay/liferay-frontend-guidelines/blob/master/dxp/dev_dependencies.md).

Reply:
Here's [the related ticket](https://github.com/liferay/liferay-npm-tools/issues/453) to enforce this via `liferay-npm-scripts`. (We do have a Java-based [JSONPackageJSONCheck](https://github.com/liferay/liferay-portal/blob/ad9a7eb092afbc7f345a66c4b0202854b4cc9fbb/modules/util/source-formatter/src/main/java/com/liferay/source/formatter/checks/JSONPackageJSONCheck.java#L73) that I would have expected to complain about this — no idea why it didn't — but we'll be able to control things in a more granular fashion if we start doing the checking from the JS side anyway.)